frontend/src/js/common/helpers/mergeRefs.ts

```diff
@@ -0,0 +1,17 @@
+type Mutable<T> = {
+  -readonly [k in keyof T]: T[k];
+};
```

Comment:
Hmmm in other implementations I can see them using a `MutableRefObject` type from React. You probably copied the code below from somewhere, right? I guess I wonder whether we shouldn't just use some external implementation like https://github.com/gregberge/react-merge-refs but up to you